wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'create',
+  'seed' => 'recreate',
```

Comment:
This is what we had, but I renamed the rake task.  This would be destructive of the whole database, and in an embedded context we don't want to do that.  The card related tables will be added and removed with migrations, not dropping and creating from the schema.